contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol

```diff
@@ -221,7 +222,8 @@ abstract contract FunctionsBilling is HasRouter, IFunctionsBilling {
     RequestBilling memory /* billing */
   ) public view override returns (uint96) {
     // NOTE: Optionally, compute additional fee here
-    return IFunctionsRouter(address(_getRouter())).getAdminFee();
+    IFunctionsRouter.Config memory config = IFunctionsRouter(address(_getRouter())).getConfig();
```

Comment:
_getRouter() already returns `IFunctionsRouter`, so no need to cast it anymore

Reply:
I tried it and _getRouter() returns an IOwnableRouter which I still need to cast to an IFunctionsRouter.

Reply:
Ahhh.. it's the RouterBase, I see I see

Reply:
Let's change `IOwnableRouter` to be `interface IOwnableRouter is IOwnable, IFunctionsRouter` if you don't mind making that quick change in this PR

Reply:
Doesn't that make it an IOwnableFunctionsRouter?

Also, this has cascading changes elsewhere where we can eliminate casting.

Reply:
I made this change and removed the unnecessary casting where I saw it.

I think further casing could be removed by making it an IOwnableFunctionsRouterAndSubscriptionSubscription, but that seemed messy.  I am hoping we can merge as-is.